test/client/fixtures/runner/automation/regression-test.js

```diff
@@ -262,10 +262,9 @@ $(document).ready(function () {
                 .addClass(TEST_ELEMENT_CLASS)
                 .click(function () {
                     clicked = true;
-                })
-                .appendTo('body');
+                });
 
-        window.setTimeout(function () {
+        $iframe[0].addEventListener('load', function () {
```

Comment:
Sorry for this, but we've fixed the `waifForIframe` function in `qunit-harness`. Please try to use it again